frontend/src/components/Cards/CardReport.tsx

```diff
@@ -0,0 +1,56 @@
+import { Card, CardDescription, CardFooter, CardTitle } from '../Card'
+import { Button } from '~/components/Button'
+import heartOutline from '~/assets/icons/heartOutline.icon.svg'
+import heartOutlineFill from '~/assets/icons/heartOutlineFill.icon.svg'
+import { useState } from 'react'
+import { ICard } from '~/types/card.type'
+
+const CardReport: React.FC<ICard & { isOutStanding: boolean }> = ({
```

Comment:
Như này là được rồi mà
```suggestion
const CardReport = ({
```